impeller/aiks/aiks_unittests.cc

```diff
@@ -3849,13 +3852,16 @@ TEST_P(AiksTest, GaussianBlurMipMapImageFilter) {
         std::max(it->texture->GetTextureDescriptor().mip_count, max_mip_count);
   }
   EXPECT_EQ(max_mip_count, blur_required_mip_count);
+  // The log is FML_DLOG, so only check in debug builds.
+#ifndef NDEBUG
```

Comment:
Do we build the unittests outside of debug mode?

Reply:
Yea, we run these in release build when we do golden image testing.  The reason why was trying to keep the visual result we are looking at as close as possible to what users would see.  It may be necessary now too that swiftshader goldens take >3s to generate on debug builds.